dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/K8s.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.dao.entity;
+
+import java.util.Date;
+
+import com.baomidou.mybatisplus.annotation.IdType;
+import com.baomidou.mybatisplus.annotation.TableField;
+import com.baomidou.mybatisplus.annotation.TableId;
+import com.baomidou.mybatisplus.annotation.TableName;
+
+/**
+ * multi-data centre k8s temporary structure, waiting for new feature to complete will switch
+ */
+@TableName("t_ds_k8s")
```

Comment:
Sorry, I can't get your point to add the `t_ds_k8s` table, and what features do you want to implement?

Reply:
Hi，you have a multi-environment feature being worked on https://github.com/apache/dolphinscheduler/issues/7623, I consulted Klein and  told me i could support multiple k8s first by db store and wait for that multi-environment to be finished,or I re-implement the transition solution along the lines of your idea, as long as it supports different k8s clusters